ci-automation/vms.sh

```diff
@@ -156,8 +156,12 @@ function _vm_build_impl() {
         mv "${CONTAINER_IMAGE_ROOT}/${arch}-usr/" "./${images_out}/"
 
     ( cd images/latest ; ln -s flatcar_production_openstack_image.img.bz2 flatcar_production_brightbox_image.img.bz2 )
+    # For the digest creation we need the vmlinuz at the same folder
+    # because the PXE vmlinuz is a symlink to it
+    mv images/latest-input/flatcar_production_image.vmlinuz images/latest/
     create_digests "${SIGNER}" "images/latest/"*
     sign_artifacts "${SIGNER}" "images/latest/"*
+    mv images/latest/flatcar_production_image.vmlinuz images/latest-input/
```

Comment:
This probably needs to move the digests and stuff out too, right? So maybe
```suggestion
    mv images/latest/flatcar_production_image.vmlinuz* images/latest-input/
```

Reply:
Thanks, I've restarted the tests

Reply:
[Timestamps](http://bincache.flatcar-linux.net/images/arm64/9999.9.9+kai-pxe-vmlinuz-sign/) tell that it is working correctly now :)